File: SEMANA 1/TALLER 1/Ejercicio2.cpp
#include <iostream>
#include <cstdlib>
#include <ctime>
using namespace std;

int main() {
    srand(time(0));

    const int N = 500;
    int pares = 0;
    int impares = 0;

    for (int i = 0; i < N; i++) {
        int valor = 50 + rand() % 51;
        if (valor % 2 == 0)
            pares++;
        else
            impares++;
    }

    cout << "Cantidad de pares: " << pares << endl;
    cout << "Cantidad de impares: " << impares << endl;

    return 0;
}

File: SEMANA 1/TALLER 1/Ejercicio2.py
import random

pares = 0
impares = 0

for _ in range(500):
    valor = random.randint(50, 100)
    if valor % 2 == 0:
        pares += 1
    else:
        impares += 1

print(f"Cantidad de pares: {pares}")
print(f"Cantidad de impares: {impares}")